Makefile

```diff
@@ -8,4 +8,8 @@ serve:
 
 dev:

```

Comment:
If you want to refactor the Makefile, I suggest also using the generate command in the dev command, like:

dev: generate
   go run main.go ...